webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -239,14 +248,27 @@ export const GlideTable: React.FC<Props> = ({
     [data, columnIds, columnDefs],
   );
 
-  const handleCellClicked = useCallback((cell: Item) => {
-    const [, row] = cell;
-    if (row === undefined) return;
-    setSelection(({ rows }: GridSelection) => ({
-      columns: CompactSelection.empty(),
-      rows: rows.hasIndex(row) ? rows.remove(row) : rows.add(row),
-    }));
-  }, []);
+  const handleCellClicked = useCallback(
+    (cell: Item) => {
+      const [col, row] = cell;
+      if (row === undefined) return;
+
+      const columnId = columnIds[col];
+      const rowData = data[row];
+      if (Loadable.isLoaded(rowData)) {
+        const cell = columnDefs[columnId].renderer(rowData.data, row) as ClickableCell;
+        if (String(cell?.data?.kind) === 'links-cell') {
```

Comment:
can this be 
```
(cell.data as LinkCellProps).kind === 'links-cell')
```
for the sake of consistency with the `isMatch` functions of the custom renderers? can also remove the custom type `ClickableCell` as well.

Reply:
This gets me to:
`Property 'data' does not exist on type 'GridCell'.   Property 'data' does not exist on type 'LoadingCell'.`

Reply:
ah okay